contributors.md

```diff
@@ -95,6 +95,7 @@ The following people are not part of the development team, but have been contrib
 * Adrian Zdanowicz (CookiePLMonster) - Misc.
 
 ## Bug fixes
+* Struan Clark (xtruan)
```

Comment:
This list is in chronological order. Please append your name at the end of the list instead.

Reply:
Moved to bottom, thanks!